src/fields/BaseOptionsField.php

```diff
@@ -390,6 +378,20 @@ public function normalizeValue(mixed $value, ?ElementInterface $element = null):
         return $value;
     }
 
+    /**
+     * Check if given option should be marked as selected.
+     *
+     * @param array $option
+     * @param mixed $value
+     * @param array $selectedValues
+     * @param bool $selectedBlankOption
+     * @return array
+     */
+    protected function isOptionSelected(array $option, mixed $value, array $selectedValues, bool $selectedBlankOption): array
+    {
+        return [in_array($option['value'], $selectedValues, true), $selectedValues, $selectedBlankOption];
+    }
```

Comment:
Given the method name, I think it would make more sense to have it return `bool`, and have `$selectedValues` and `$selectedBlankOption` passed by reference, in case they need to be modified.

```suggestion
    protected function isOptionSelected(array $option, mixed $value, array &$selectedValues, bool &$selectedBlankOption): array
    {
        return in_array($option['value'], $selectedValues, true);
    }
```

Reply:
Makes sense! Adjusted.